src/wireviz/Harness.py

```diff
@@ -170,7 +170,7 @@ def create_graph(self) -> Graph:
                      f'{cable.wirecount}x' if cable.show_wirecount else None,
                      f'{cable.gauge} {cable.gauge_unit}{awg_fmt}' if cable.gauge else None,
                      '+ S' if cable.shield else None,
-                     f'{cable.length} m' if cable.length > 0 else None,
+                     f'{cable.length} {cable.lengthunit}' if cable.length > 0 else None,
```

Comment:
I can see an unfortunate difference in naming style between these two attribute pairs:
- `cable.gauge` and `cable.gauge_unit`
- `cable.length` and `cable.lengthunit`

Feel free to discuss naming conventions in a new issue, but I suggest we try to keep it consistent.

Reply:
added the underbar to length_unit in #171